pulp_smash/tests/rpm/api_v2/test_download.py

```diff
@@ -84,7 +77,7 @@ def setUpClass(cls):
 
         # Corrupt an RPM. The file is there, but the checksum isn't right.
         cli_client = cli.Client(cls.cfg)
-        sudo = '' if _is_root(cls.cfg) else 'sudo '
+        sudo = '' if utils.is_root(cls.cfg) else 'sudo '
```

Comment:
Seeing this sudo/prefix defined in more than one place made me wonder if `utils.sudo()` would be a good thing to have.